utils/pmdk.spec.in

```diff
@@ -24,17 +24,14 @@
 %bcond_with fabric
 %endif
 
-# ndctl v59.2 is available on:
+# ndctl v60.0 is available on:
 #   openSUSE Tumbleweed, Leap >=42.3; SLE >=12 SP3
-#   Fedora >=26; RHEL >=7.6
-#%if (0%{?suse_version} > 1500) || (0%{?sle_version} >= 120300) || (0%{?fedora} >= 26) || (0%{?rhel} >= 7)
-#%bcond_without ndctl
-#%else
-#%bcond_with ndctl
-#%endif
-
-# XXX: by default build w/o ndctl, unless explicitly enabled
+#   Fedora >=28; RHEL >=7.6
```

Comment:
But you later check for Fedora>=26...

https://koji.fedoraproject.org/koji/packageinfo?packageID=22268 says ndctl 60.1 is available for Fedora>=26.